src/index.html

```diff
@@ -13,6 +13,54 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="star__blok">
+      <div class="star__line stars--0">
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+      </div>
+
+      <div class="star__line stars--1">
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+      </div>
+
+      <div class="star__line stars--2">
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+      </div>
+
+      <div class="star__line stars--3">
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
```

Comment:
The class name 'srar__style' should be corrected to 'star__style' for consistency and to ensure the CSS styles are applied as expected.